common/protob/pb2py

```diff
@@ -714,7 +714,11 @@ def main(
     if verbose:
         logging.basicConfig(level=logging.DEBUG)
 
-    descriptor_proto = protoc(proto)
+    # The proto tuple is sorted to give consistent results in different environments
+    list_proto = list(proto)
+    list_proto.sort()
```

Comment:
This is needed, because other environments (eg. my local system) can have different default sorting of files:

- On my system messages.proto are **before** messages-stellar.proto.
- In CI environment, the messages.proto are **after** messages-stellar.proto.

This leads to a different order of generated classes in messages.py (and the order of enums in other .py file). The change of order has no impact on functionality, but it is captured as "diff" during Prebuild Gen check resulting in check fail. Sorting the file names consistently using python sort prevents the issue.

Reply:
Nit: could be:
```
descriptor_proto = protoc(sorted(proto))
```

Reply:
Fixed here: https://github.com/trezor/trezor-firmware/pull/4265/commits/adaa01552daed7348ac81c6ab430a058e9bddf1d